modules/std.lobster

```diff
@@ -152,8 +152,14 @@ def insert_ordered(xs, x, lt):
     xs.push(x)
 
 def randomize(xs):
-    // Simple and effective, but remove makes it expensive.
-    for(xs.length) i: xs.push(xs.remove(rnd(xs.length - i)))
+    for(xs) x, i:
+        if i:
+            let j = rnd(i+1)
+            if j != i:
```

Comment:
given that these values are usually unequal, and that the code below is neutral to them being the same, it be cheaper to leave this `if` out?